gitea/Dockerfile

```diff
@@ -0,0 +1,77 @@
+#============================#
+#  ALEXBELGIUM'S DOCKERFILE  #
+#============================#
+#           _.------.
+#       _.-`    ('>.-`"""-.
+# '.--'`       _'`   _ .--.)
+#    -'         '-.-';`   `
+#    ' -      _.'  ``'--.
+#        '---`    .-'""`
+#               /`
+#=== Home Assistant Addon ===#
+
+#################
+# 1 Build Image #
+#################
+
+ARG BUILD_FROM=gitea/gitea
+
+FROM ${BUILD_FROM}
+
+##################
+# 2 Modify Image #
+##################
+
+ENV USER git
+ENV GITEA_CUSTOM /share/gitea/data
```

Comment:
I'm sorry but actually I think this could create some more issues as gitea would have to create the folder by itself, but not sure the `git` user will have mkdir permission on `/share`. An alternative is to add a `sed -i "1a mkdir -p /share/gitea/data" /usr/bin/entrypoint` instruction to Dockerhub.

The last solution : I see that the default gitea container location `ENV GITEA_CUSTOM=/data/gitea` would store the data in `/data` which would also allow remanence

Reply:
perfect, i will remove my custom `ENV GITEA_CUSTOM`